### PR TITLE
Example activity

### PR DESCRIPTION
**Pull request by LukasPaczos**
This PR adds the example activity.

gestures test app

### CODE REVIEW

**Comment by zugaldia** [comment]
Once you merge https://github.com/mapbox/mapbox-gestures-android/pull/5, could you rebase to see if we can make this PR pass CI?

File: app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import android.app.DialogFragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+public class HelpDialogFragment extends DialogFragment {
+  public static final String TAG = "help_dialog_fragment_tag";
```
Comment (tobrun) [inline app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java]: can be private?

File: app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  public HelpDialogFragment() {
+    // Required empty public constructor
+  }
+
+  public static HelpDialogFragment newInstance() {
+    HelpDialogFragment fragment = new HelpDialogFragment();
+    return fragment;
```
Comment (tobrun) [inline app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java]: if no arguments are set you can just do `return new HelpDialogFragment()`?

File: app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+    if (enabled) {
+      velocityEnabled = true;
+      velocityButton.setText(R.string.velocity_on);
+    } else {
+      velocityEnabled = false;
+      velocityButton.setText(R.string.velocity_off);
+    }
+  }
+
+  private class ExclusiveObject {
```
Comment (tobrun) [inline app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java]: Don't really understand the concept of the class from the name: it excludes objects?

File: app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+          return "Scale";
+        case AndroidGesturesManager.GESTURE_TYPE_SHOVE:
+          return "Shove";
+        case -1: // (none)
+        default:
+          return "none";
+      }
+    }
+
+    Set<Integer> getExclusivesList() {
```
Comment (tobrun) [inline app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java]: returning a `Set` from a method name that has the name `List`

File: app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java (hunk 1 above)
Comment (LukasPaczos) [reply app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java]: Used in the Activity, can be package private.

File: app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java (hunk 3 above)
Comment (LukasPaczos) [reply app/src/main/java/com/mapbox/android/gestures/testapp/MainActivity.java]: The purpose of the class is to provide the `String` representation of the gestures for the `Spinner`. I'll try with `ExclusiveSetSpinnerObject` to make it clearer.

**Pull request merged**